vllm/engine/arg_utils.py

```diff
@@ -434,6 +438,29 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                             'MoE layers will be sharded according to the '
                             'product of the tensor-parallel-size and '
                             'data-parallel-size.')
+        parser.add_argument('--data-parallel-size-local',
+                            '-dpl',
+                            type=int,
+                            default=EngineArgs.data_parallel_size_local,
+                            help='Number of data parallel replicas to run on '
+                            'this node.')
+        parser.add_argument('--data-parallel-start-rank',
+                            '-dpr',
+                            type=int,
+                            default=EngineArgs.data_parallel_start_rank,
+                            help='Starting data parallel rank for secondary '
+                            'nodes.')
```

Comment:
these two args should go to `vllm/entrypoints/cli/serve.py` since they are for online serving only.

Reply:
@youkaichao I moved the start-rank one but the other one is used within the AsyncLLM impl which in theory could be used in other contexts (e.g. different front-end), so it makes sense for it to be in the config I think.

I could move it from `EngineArgs` into `AsyncEngineArgs` but then it would be on it's own so thought better to keep with the other data parallel args.